src/spacingAliasesProvider.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import * as React from 'react'
+import PropTypes from 'prop-types'
+
+export type SpacingAliases = {
+  [spacingAlias: string]: number,
+}
+
+type Props = {
+  spacingAliases: SpacingAliases,
+  children: React$Node,
+}
+
+export const SpacingAliasesContext = { spacingAliases: PropTypes.object }
```

Comment:
I think we should namespace this to something like `xnReflexSpacingAliases`, or `xnReflex: { spacingAliases }` since this is a global namespace we are tapping into.

Reply:
This shouldn't be specific to spacing though the idea is that we'll use the provider for any other variables (number of columns, breakpoints...)

Reply:
I don't think this should be prefixed with xn.  What about renaming to ConfigProvider?

Reply:
I want something specific to the library name, `xn-reflex`

Reply:
Oh I see what you're saying.